content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -2,24 +2,67 @@
 Multiple sales teams
 ====================
 
-Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
-their own unique sales processes.
+The *Multi Teams* feature in Odoo's *CRM* app allows salespeople to be assigned to multiple sales
+teams. Multiple teams can be created and managed in one dashboard, each with their own pipeline for
+leads and opportunities.
 
+Enable multi teams
+==================
 
-Create a new sales team
+To allow salespeople to be assigned to more than one sales team, the :guilabel:`Multi Teams` setting
+needs to be enabled. First, navigate to :menuselection:`CRM app --> Configuration --> Settings`.
+Under the :guilabel:`CRM` section, tick the checkbox labeled :guilabel:`Multi Teams`. Then, click
+:guilabel:`Save` at the top-left of the page.
+
+.. image:: multi_sales_team/multi-team-setting.png
+   :align: center
+   :alt: The Multi Teams setting in the CRM app.
+
+.. _crm/assignment-rules:
+
+Enable assignment rules
 =======================
 
-To create a new sales team, go to :menuselection:`CRM --> Configuration --> Sales Teams`, then
-click :guilabel:`Create`.
+To periodically :doc:`assign leads <../track_leads/lead_scoring>` based on rules, the
+:guilabel:`Rule-Based Assignment` feature needs to be enabled. Navigate to :menuselection:`CRM app
+--> Configuration --> Settings`. Under the :guilabel:`CRM` section, tick the checkbox labeled
+:guilabel:`Rule-Based Assignment`.
+
+This causes a new field, :guilabel:`Running`, to appear. Select an option from the drop-down menu in
+this field to manage how assignments are triggered.
+
+The :guilabel:`Rule-Based Assignment` feature can be set to run :guilabel:`Manually`, meaning an
```

Comment:
Suggestion for specificity: 
It's implied right here to run **Manually** on the **Settings > CRM** page, however I would clearly specify where it's possible to run the rules manually, and how to do it, for users who wish to do this.